docs/devguide/changelog.rst

```diff
@@ -2,6 +2,73 @@ Changelog
 =========
 
 
+Parsl 1.0.0
+-----------
+
+Released on June <FILL>th, 2020
+
+Parsl v1.0.0 includes 57 closed issues and 232 pull requests with contributions (code, tests, reviews and reports) from:
+
+Akila Ravihansa Perera @ravihansa3000, Aymen Alsaadi @AymenFJA, Anna Woodard @annawoodard,
+Ben Clifford @benclifford, Ben Glick @benhg, Benjamin Tovar @btovar, Daniel S. Katz @danielskatz,
+Daniel Smith @dgasmith, Douglas Thain @dthain, Eric Jonas @ericmjonas, Geoffrey Lentner @glentner,
+Ian Foster @ianfoster, Kalpani Ranasinghe @kalpanibhagya, Lindsey Gray @lgray, Logan Ward @WardLT,
+Lyle Hayhurst @lhayhurst, Mihael Hategan @hategan, Rajini Wijayawardana @rajiniw95, @saktar-unr,
+Tim Shaffer @trshaffer, Tom Glanzman @TomGlanzman, Yadu Nand Babuji @yadudoc and,
+Zhuozhao Li @ZhuozhaoLi
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+* `WorkQueueExecutor`: a new executor that integrates functionality from `Work Queue <http://ccl.cse.nd.edu/software/workqueue/>`_ is now available.
+
+  * WorkQueueExecutor is now in Beta.
+  * WorkQueueExecutor now supports function tagging and resource specification
+  * Support for resource specification kwarg `issue#1675 <https://github.com/Parsl/parsl/issues/1675>`_
+
+* `HighThroughputExecutor` will expose worker information via environment variables: `PARSL_WORKER_RANK` and `PARSL_WORKER_COUNT`
+* Separate htex worker log directories by block id `issue#1508 <https://github.com/Parsl/parsl/issues/1508>`_
+* Remove App decorator deprecated in 0.8 `issue#1539 <https://github.com/Parsl/parsl/issues/1539>`_
+* Significant progress in an ongoing effort to add type-checking to Parsl internal components.
```

Comment:
This is written more as what we did than as functionality.  Maybe "Limited type-checking in Parsl internal components (full type-checking will be implemented in v1.1.0)"

Reply:
make promises about the future in the roadmap, not in the history document, especially when they're promises that won't come true

Reply:
Ok, then "Limited type-checking in Parsl internal components (as part of an ongoing effort)"